README.md

```diff
@@ -1561,6 +1561,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
 - [FreedomBox](https://wiki.debian.org/FreedomBox) - Community project to develop, design and promote personal servers running free software for private, personal, communications. `GPL-3.0` `Python/Other`
 - [FreeNAS](https://www.freenas.org/) - Network-attached storage (NAS) software based on FreeBSD and the OpenZFS file system. Support for SMB, AFP, NFS, iSCSI, SSH, rsync and FTP/TFTP protocols. Advanced features include full-disk encryption and plug-ins. ([Source Code](https://github.com/freenas/freenas)) `BSD-3-Clause` `Python/Other`
 - [HomelabOS](https://gitlab.com/NickBusey/HomelabOS) - Your very own offline-first privacy-centric open-source data-center. `MIT` `Docker`
+- [Micro](https://github.com/micro/micro) - A distributed systems runtime for the cloud and beyond. A fully vertically integrated cloud-native development platform focused entirely on the developer experience. `Apache 2.0` `Go`
```

Comment:
Please link to https://m3o.com/about.html as the main link, add sue the Github URL as _Source code_ link

Reply:
The website is related to the company rather than the project and that url may change. I linked to github as I had seen the link above do the same even though they have a website.

Reply:
I have linked directly to the homepage as that seemed more appropriate